src/garage/envs/task_id_wrapper.py

```diff
@@ -0,0 +1,54 @@
+"""Wrapper for Metaworld environment to provide properly-named tasks."""
+
+import gym
+
+
+class TaskIdWrapper(gym.Wrapper):
```

Comment:
if this only works for metaworld, please put that in the name `MetaworldTaskIdWrapper`